spec/seafoam/command_spec.rb

```diff
@@ -127,7 +128,7 @@
 
   describe '#props' do
     it 'prints properties for a file' do
-      @commands.send :props, @fib_java.to_s
+      @commands.send :props, @fib_java
```

Comment:
Is this change intentional? It doesn’t look related.

Reply:
Unrelated tidy-up - `@fib_java` is already a `String`!